docs/confidential-ledger-setup-python.md

```diff
@@ -0,0 +1,155 @@
+## Use the python SDK to setup a confidential ledger
+
+First import the required libraries:
+
+For more information on these libraries and how to install them, see the [confidential ledger documentation](https://docs.microsoft.com/en-us/azure/confidential-ledger/quickstart-python?tabs=azure-cli)
+
+```
+from dotenv import load_dotenv
+import os
+
+from azure.identity import ClientSecretCredential
+from azure.mgmt.confidentialledger import ConfidentialLedger as ConfidentialLedgerAPI
+from azure.mgmt.confidentialledger.models import ConfidentialLedger
+from azure.confidentialledger import ConfidentialLedgerClient
+from azure.confidentialledger.identity_service import (
+    ConfidentialLedgerIdentityServiceClient,
+)
+```
+
+If using a .env file you can run:
+
+```
+load_dotenv()
+
+# Set tenant ID from .env file
+client_id = os.getenv("CL_APP_ID")
+# scan:ignore
```

Comment:
Can probably remove this from the docs, as this is just our code scan tool

Reply:
good call, I'll remove before merging